### ISSUE TITLE
Question on the mobileNetV2 convergence time.

### ISSUE THREAD

**Issue by jianyinglang** [tensorflow/models#9719]
I found this information in https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet/README.md <The numbers above can be reproduced using slim's train_image_classifier. Below is the set of parameters that achieves 72.0% for full size MobileNetV2, after about 700K when trained on 8 GPU. If trained on a single GPU the full convergence is after 5.5M steps. Also note that learning rate and num_epochs_per_decay both need to be adjusted depending on how many GPUs are being used due to slim's internal averaging.>

I wonder what is run time to achieve this convergence?
